blog/en/blog/2022/07/21/weekly-report-0715.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Biweekly Report (Jul 1 - Jul 15)
+keywords:
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
+tags: [Events]
+---
+
+> From Jul 1st to Jul 15th, 28 contributors submitted 99 commits for Apache APISIX. Thank you all for your contributions to Apache APISIX. It is your selfless contribution to make the Apache APISIX project better!
+
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX has grown as a community from the first day of open source and has quickly become the most active open source API gateway project in the world. These achievements cannot be achieved without the joint efforts of our community partners.
+
+"If you want to go fast, go alone.If you want to go far, go together." The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
+
+We have also compiled some issues suitable for newcomers to the community to participate in! If you are interested, don't miss it!
+
+## Contributor Statistics
+
+![Contributors List](https://static.apiseven.com/2022/weeklyreport/0721/4.jpg)
+
+![New Contributors](https://static.apiseven.com/2022/weeklyreport/0721/2.png)
```

Comment:
Borbur already started contributed on or before June 30th, see https://apisix.apache.org/blog/2022/06/21/weekly-report-0621/#contributor-statistics.

Why is he listed as 'New contributor' here?